src/functions/getTestTypesById.ts

```diff
@@ -12,10 +13,14 @@ export const getTestTypesById: Handler = (event, context, callback) => {
   // Validate query parameters
   const queryParamSchema = Joi.object().keys({
     fields: Joi.string().regex(/^(testTypeClassification|defaultTestCode|linkedTestCode),?\s*((testTypeClassification|defaultTestCode|linkedTestCode),?\s*)?((testTypeClassification|defaultTestCode|linkedTestCode),?\s*)?$/).required(),
-    vehicleType: Joi.any().only([ "psv", "hgv", "trl" ]).required(),
-    vehicleSize: Joi.any().only([ "small", "large" ]),
-    vehicleConfiguration: Joi.string().allow(null).optional(),
-    vehicleAxles: Joi.number().min(0).max(99).allow(null).optional()
+    vehicleType: Joi.string().only(Object.values(ForVehicleType)).required(),
+    vehicleSize: Joi.string().only(Object.values(ForVehicleSize)),
+    vehicleConfiguration: Joi.string().only(Object.values(ForVehicleConfiguration)).allow(null),
```

Comment:
Should this not be optional still?

Reply:
It is still optional. Joi considers all keys as [optional by default](https://hapi.dev/family/joi/api/?v=17.1.0#anyoptional). I removed these to have a cleaner situation.

Reply:
default is optional